sympy/plotting/plot.py

```diff
@@ -32,6 +32,8 @@
 from sympy.core.compatibility import range, Callable
 from sympy.utilities.iterables import is_sequence
 from .experimental_lambdify import (vectorized_lambdify, lambdify)
+from sympy import fraction
+from sympy import real_roots
```

Comment:
You can add these to the main sympy import [above](https://github.com/sympy/sympy/pull/18856/files#diff-e4ca0e0fed60e3bd29d26591f5ba3f88R29)

Reply:
Yeah sure I will this